### ISSUE TITLE
Missing support for adjustment based installed effect.

### ISSUE THREAD

**Issue by stubbis88** [joggs/home_assistant_ebeco#32]
If I set the power to 50% this shows as 50 °C..

**Comment by PerfectlyNormal** [comment]
Hey

Sorry for the late reply. I'm a bit unsure how to reproduce this. Where do you set the power to 50%? Is it on the physical thermostat, or in HomeAssistant?

**Comment by stubbis88** [comment]
Yes, this is a setting on physical thermostat if you don't have an floor sensor.. typical on the bathroom.
